include/clang/Interpreter/CppInterOp.h

```diff
@@ -331,6 +335,16 @@ namespace Cpp {
   CPPINTEROP_API bool ExistsFunctionTemplate(const std::string& name,
                                              TCppScope_t parent = 0);
 
+  CPPINTEROP_API std::vector<TCppFunction_t>
+  GetTemplatedMethods(const std::string& name, TCppScope_t parent = 0,
+                      const std::string& filter = "");
+
+  CPPINTEROP_API TCppIndex_t GetNumTemplatedMethods(TCppScope_t scope,
+                                                    bool accept_namespace);
+
+  CPPINTEROP_API std::string GetTemplatedMethodName(TCppScope_t scope,
+                                                    TCppIndex_t imeth);
```

Comment:
Both functions are redundant and can be implemented with the existing API on the caller's side.

Reply:
This will require extra code to extract the count in CPyCppyy's dispatcher and proxywrapper

https://github.com/compiler-research/CPyCppyy/blob/8642409667a8bfd2acc2d3e0ba2e964efb937ac4/src/ProxyWrappers.cxx#L300

I thought I should keep changes on CPyCppyy minimal?

Reply:
Yes, you can provide that function in the cppyy-backend, not CppInterOp.

Reply:
done